Cangjie5.txt

```diff
@@ -6298,6 +6297,7 @@
 矍	buoge
 𬑠	buoge
 瞧	buogf
+𪾷	buogs
```

Comment:
`buogs` is significantly different to `bhuob`, 

@Jackchows please double check if `buogs` is correct, thank you!

Reply:
You are right.